### ISSUE TITLE
Browsers connecting through a VPN don't seem to be able to connect

### ISSUE THREAD

**Issue by haydenmc** [Glimesh/janus-ftl-plugin#22]

**Comment by danstiner** [comment]
I am able to connect through WireGuard, I suspect this depends on the network and exact VPN tech.

Likely in some cases when the VPN encapsulates video packets they become larger than the MTU of some router in the path and packet loss happens.

Lowering the packet size is the only solution and doing that requires either a change in ftl-sdk or transcoding.

**Comment by haydenmc** [comment]
From what I've seen, this is due to some VPNs actually killing the ICE exchange to avoid "leaking" the user's IP address

**Comment by danstiner** [comment]
Ah yeah, totally some do that, I think often through a browser extension? Nothing we can do about that right except ask them to disable it or fallback to HLS.

**Comment by haydenmc** [comment]
Yeah, HLS might be the answer here unless TURN helps out...? I vaguely remember that being a possibility. 😅
